product_images_downloader.py

```diff
@@ -122,47 +111,47 @@ def main() :
 
     details = input_details()
 
-    if details == -1 :
+    if details == -1:
         sys.exit('Exiting Since product name is not entered !!')
-        
-    item_name, num_prods, num_pages = details[0], details[1], details[2]
-
 
+    item_name, num_pages = details[0], details[1]
+    
     ama_search = Amazon()  # instantiate amazon search class
     pg_src = ama_search.search_text(item_name)
 
+    pages_traversed = 0
     count_products = 0
-    pages_traversed = 1   # already 1 page is opened !!!
 
     # break when required products count is reached or req of pages are traversed !!!
 
-    while (count_products < num_prods) and (pages_traversed < num_pages ):  
-        
+    while (pages_traversed < num_pages):
+

```

Comment:
* Would not have expected the loop to be "Check these many pages, or these many items, whichever comes first".
* Loop did not work for "Just do one page

Updates: Removed product count entirely, now ONLY looping through however many pages.